### PR TITLE
Release 0.87

### PR DESCRIPTION
**Pull request by jpknwls**

### CODE REVIEW

File: vault/changelog.md [hunk 1]
```diff
@@ -2,11 +2,36 @@
...
+### Features
+- feat(vaults): early implementation of self contained vaults ([[docs|dendron://dendron.dendron-site/dendron.topic.vaults.self-contained]]) (#2517) @kaan
+- feat(workspace): doctor command to find (and suggest fix for) keyboard conflicts ([[docs|dendron://dendron.dendron-site/dendron.topic.doctor]]) (#2578) @hikchoi
+
+### Enhancements
+- enhance(workspace): faster preview refresh (#2527) @tuling
+- enhance(workspace): faster editing in large workspaces (#2527) @tuling
+- enhance(internal): provide a standard way to create and read backup files ([[docs|dendron://dendron.dendron-site/dendron.ref.commands#open-backup]]) (#2521) @hikchoi
+- enhance(workspace): warn on keyboard conflicts on first install (#2578) @hikchoi
+- enhance(pods): support exporting a note to multiple Airtable destination ([[docs|dendron://dendron.dendron-site/dendron.topic.pod-v2.ref.builtin.airtable.export]]) (#2582) @Joshi
```
Comment (ScriptAutomate) [inline vault/changelog.md:21]:
```suggestion
- enhance(pods): support exporting a note to multiple Airtable destinations ([[docs|dendron://dendron.dendron-site/dendron.topic.pod-v2.ref.builtin.airtable.export]]) (#2582) @Joshi
```

File: vault/changelog.release.2022-03-22.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+- fix(workspace): fix crash that can happen when bad frontmatter is present during doc save
+- fix(publish): customHeaderPath breaks publishing if value is set to anything except `header.html`
+- fix(basics): ensure note title is always a string to avoid errors
+- fix(sync): better error message on `Workspace Add and Commit`
+- fix(lookup): lookup (without spaces) should be case-insensitive
+- fix(workspace): issue with notes not being saved on export
+- fix(workspace): race condition when backing up configuration
+- feat(workspace): doctor command to find (and suggest fix for) keyboard conflicts
+- enhance(workspace): warn on keyboard conflicts on first install
+- enhance(pods): support exporting a note to multiple Airtable destination
```
Comment (ScriptAutomate) [inline vault/changelog.release.2022-03-22.md:35]:
```suggestion
- enhance(pods): support exporting a note to multiple Airtable destinations
```

File: vault/changelog.release.2022-03-22.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+- fix(workspace): prevent errors in Open Backup Command and Run Migration Command in native workspaces
+
+
+## Community
+
+### General Updates
+Check out our founder, Kevin, discuss his background and how he started Dendron on the *From Zero to One* podcast! Find out more details in the [episode blurb](https://www.0011.one/posts/S3E10) and listen on [Spotify](https://open.spotify.com/episode/5JhPoOCvXl3om7m905dhC4?si=cf6240803a4940e7&nd=1) and [Xiaoyuzhoufm](https://www.xiaoyuzhoufm.com/episode/621f25f2761e0799b628feca).
+
+
+
```
Comment (ScriptAutomate) [inline vault/changelog.release.2022-03-22.md:47]:
```suggestion
```

File: vault/changelog.release.2022-03-22.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+- fix(workspace): issue with notes not being saved on export
+- fix(workspace): race condition when backing up configuration
+- feat(workspace): doctor command to find (and suggest fix for) keyboard conflicts
+- enhance(workspace): warn on keyboard conflicts on first install
+- enhance(pods): support exporting a note to multiple Airtable destination
+- fix(views): unblock preview rendering when backlink is invalid
+- fix(workspace): race condition when loading native workspace
+- fix(workspace): prevent errors in Open Backup Command and Run Migration Command in native workspaces
+
+
```
Comment (ScriptAutomate) [inline vault/changelog.release.2022-03-22.md:40]:
```suggestion
```

**Review by ScriptAutomate — APPROVED** [review]

**Review by ScriptAutomate — CHANGES_REQUESTED** [review]
We will need to list a breaking change related to the Airtable export pod update. More information coming shortly.

**Review by kevinslin — APPROVED** [review]

**Review by ScriptAutomate — APPROVED** [review]

**Pull request merged**